TombEngine/Objects/TR4/Entity/tr4_big_scorpion.cpp

```diff
@@ -245,20 +270,26 @@ namespace TEN::Entities::TR4
 						CreatureEffect2(item, BigScorpionBite1, 10, item->Pose.Orientation.y - ANGLE(180.0f), DoBloodSplat);
 					}
 					else
+					{
 						CreatureEffect2(item, BigScorpionBite2, 10, item->Pose.Orientation.y - ANGLE(180.0f), DoBloodSplat);
+					}
 
 					creature->Flags = 1;
 
 					if (creature->Enemy->IsLara() && creature->Enemy->HitPoints <= 0)
 					{
-						CreatureKill(item, BSCORPION_ANIM_KILL, 0, BSCORPION_STATE_KILL, LS_DEATH); // TODO: add big_scorpion lara extra state enum
+						CreatureKill(item, BSCORPION_ANIM_KILL, LEA_BIG_SCORPION_DEATH, BSCORPION_STATE_KILL, LS_DEATH);
 						creature->MaxTurn = 0;
 						return;
 					}
 				}
 
 				break;
 
+			case BSCORPION_STATE_KILL:
+				creature->MaxTurn = 0;
```

Comment:
As above

Reply:
Same explanation as above. :D

To ensure the Scorpion is facing at Lara while it kills her with its pincers.